redis_json/src/commands.rs

```diff
@@ -588,17 +594,32 @@ pub fn json_mget<M: Manager>(manager: M, ctx: &Context, args: Vec<RedisString>)
 
     args.last().ok_or(RedisError::WrongArity).and_then(|path| {
         let path = Path::new(path.try_as_str()?);
-        let keys = &args[1..args.len() - 1];
 
-        let format_options = FormatOptions {
-            resp3: is_resp3(ctx),
-            ..Default::default()
+        let is_resp3 = is_resp3(ctx);
+        let mut format_options = FormatOptions::new(is_resp3);
+
+        // Check if the optional argument [FORMAT STRING|JSON|EXPAND] is present
+        let keys = if is_resp3 {
+            let format = &args[args.len() - 3..args.len() - 1];
+            if let Ok(s) = format[0].try_as_str() {
+                if s.eq_ignore_ascii_case("FORMAT") {
+                    format_options.format = Format::from_str(format[1].try_as_str()?)?;
```

Comment:
Same comment as above, where do you check the `format[1]` exists?

Reply:
yes it has to exist since the command has more than 3 arguments